crates/config/src/sections/upstream_oauth2.rs

```diff
@@ -565,4 +565,11 @@ pub struct Provider {
     /// Orders of the keys are not preserved.
     #[serde(default, skip_serializing_if = "BTreeMap::is_empty")]
     pub additional_authorization_parameters: BTreeMap<String, String>,
+
+    /// Whether the login_hint should be forwarded to the provider in the
```

Comment:
One thing I just thought of, can you please update the reference documentation here? https://github.com/element-hq/matrix-authentication-service/blob/main/docs/reference/configuration.md#upstream_oauth2providers

Reply:
Done in https://github.com/element-hq/matrix-authentication-service/pull/4512/commits/e3c7b8054c670ea5fc6e0d1cf62d60aca499167a